### ISSUE TITLE
Use of rng('shuffle')

### ISSUE THREAD

**Issue by justbennet** [NISOx-BDI/SwE-toolbox#86]
We are using MATLAB R2018a and
```
SwE-toolbox$ git log | head -1
commit 622d715efd1618e3da76db8233616c24a02b2c31
```

It seems that SPM uses `rand('state', 0)`, where 0 can be replaced by any value, and that now causes an error from swe_seed() when it calls `rng('shuffle')` at line 37.

```
>> swe_seed            %  Test with no SPM use ; succeeds
>> rand('state', 0)    %  Run rand() as is done in, e.g., ./spm_coreg.m
>> swe_seed            %  Now swe_seed() fails
Error using rng (line 109)
The current random number generator is the legacy generator.  This is because
you have executed a command such as rand('state',0), which activates MATLAB's
legacy random number behavior.  You may not use RNG to reseed the legacy random
number generator.

Use rng('default') to reinitialize the random number generator to its startup
configuration, or call RNG using a specific generator type, such as
rng(seed,'twister').

Error in swe_seed (line 37)
      rng('shuffle');
 
>> clear           % Clear definition of swe_seed() ; change rng('shuffle') to rng('shuffle', 'twister')
>> swe_seed   % Now swe_seed() succeeds
```

I am not sure how sensitive your code is to the random number generation, so I was reluctant to submit a PR.  If you would like one with this selection, I will submit.

**Comment by nicholst** [comment]
Thanks for this! I had no idea how complex Matlab's random number generators had become.

I will make a fix right now, but the solution is as suggested by the error:  Insert this at the current swe_seed.m line 37:
```matlab
rng('default')
```

**Comment by nicholst** [comment]
The point of swe_seed is to give *randomly* different wild bootstrap results each time SwE/Matlab is started fresh.  SPM's use of `rand('state', 0)` in contrast is trying to give  replicable results every time *but* as a side effect activates the legacy random number generator.

So with the addition of `rng('default')` I am reactivating the default ('twister') random number generator and then setting it randomly.  That it also sets the default seed is irrelevant as the next line is `rng('shuffle')`.

I tried `rng('shuffle', 'default')` but that is an error.  (I guess I could do `rng('shuffle', 'twister')` but I'd just like to avoid dependence on peculiarities of the current Matlab rng naming/implementation.)

**Comment by justbennet** [comment]
I understand not wanting to depend on peculiarities in MATLAB.  I checked
on our version, and the two step method seems to work fine.  I see you've
put in a PR.  Thanks, as always, for the quick response.  I will add the PR
fix to our local copy.

**Comment by nicholst** [comment]
This was addressed in PR #62.
